internal/pkg/service/buffer/store/model/conditions.go

```diff
@@ -33,7 +33,7 @@ func DefaultImportConditions() Conditions {
 	}
 }
 
-func (c Conditions) Evaluate(now time.Time, s Stats) (bool, string) {
+func (c Conditions) Evaluate(now time.Time, openedAt time.Time, s Stats) (bool, string) {
```

Comment:
Tu bola logicka chyba, .... cas sa musi kontrolovat odkedy bola slice/file otvorena, ... nie od posledneho prijateho record, to by sa nemusela nikdy uzavriet.